lib/worker_manager_test.go

```diff
@@ -198,9 +201,14 @@ func TestMultiplePendingHeartbeats(t *testing.T) {
 
 	replyTime := time.Now()
 	manager.clock.(*clock.Mock).Set(replyTime)
-	offlined, onlined = manager.Tick(ctx, msgSender)
-	require.Empty(t, offlined)
-	require.Len(t, onlined, 2)
+
+	var totalOnlined int
+	require.Eventually(t, func() bool {
+		offlined, onlined = manager.Tick(ctx, msgSender)
```

Comment:
`onlined` means justOnlined, very misleading, please add comment for manager.Tick later